### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by KristinaGru**

### CODE REVIEW

File: javascript/javascript2/week3/fastestPresser/JS/fastestPresser.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+const playerTwo = document.getElementById('player-two');
+const winner1 = document.querySelector('.canvas1');
+const winner2 = document.querySelector('.canvas2');
+const gameSection1 = document.querySelector('.game-section');
+const gameSection2 = document.querySelector('.game-section:nth-child(2)');
+let sScore = 0;
+let lScore = 0;
+
+function keepScore(e) {
+  e.key === 's' ? sScore++ : e.key === 'l' ? lScore++ : null;
```
Comment (dinythomas89) [inline javascript/javascript2/week3/fastestPresser/JS/fastestPresser.js:12]: Be careful with ternary operator when you have so many conditions

File: javascript/javascript2/week3/warmup/JS/functions.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  console.log('double click!');
+});
+
+// 9.
+function jokeCreator(boolean, callbackTrue, callbackFalse) {
+  if (boolean) {
+    callbackTrue();
+  } else {
+    callbackFalse();
+  }
```
Comment (dinythomas89) [inline javascript/javascript2/week3/warmup/JS/functions.js:86]: Maybe ternary operator here, since it has only one if condition

**Review by dinythomas89 — APPROVED** [review]

File: javascript/javascript2/week3/warmup/JS/functions.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript2/week3/warmup/JS/functions.js:86]: Hope you got familiar with DOM, callbacks, array and its methods, objects

Comment (KristinaGru) [reply javascript/javascript2/week3/warmup/JS/functions.js:86]: I sure did! Appreciate all your time and effort while reviewing ❤️

**Pull request merged**